docs/database/cosmosdb.md

```diff
@@ -338,6 +341,7 @@ Explaining bucket configuration:
 |---|---|---|---|---|---|
 | [`db.collection.name`](/docs/attributes-registry/db.md) | string | Cosmos DB container name. [1] | `public.users`; `customers` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 | [`db.cosmosdb.consistency_level`](/docs/attributes-registry/db.md) | string | Account or request [consistency level](https://learn.microsoft.com/azure/cosmos-db/consistency-levels). | `Eventual`; `ConsistentPrefix`; `BoundedStaleness`; `Strong`; `Session` | `Conditionally Required` If available. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`db.cosmosdb.regions_contacted`](/docs/attributes-registry/db.md) | string | List of regions contacted by the client during a given operation. | `North Central US,Central US`; `East US 2,North Central US` | `Conditionally Required` If available | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
how would someone use this? would you filter by a certain combination? Check which combinations result in the highest latency? Does the order matter?

Reply:
In a metric, we can compare latencies for records where the dimension is set to either 'Region1' alone or 'Region1, Region2.' If the metric shows a latency value for 'Region1, Region2,' it indicates that the SDK is making cross-regional calls, which results in higher latency for the customer."
No order doesn't matter there.